src/Ardalis.ApiEndpoints/Ardalis.ApiEndpoints.csproj

```diff
@@ -25,6 +25,10 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.1.0" Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' != 'netstandard2.1' ">
+    <Compile Remove="FluentGenerics\EndpointBaseASync.Streams.cs" />
+  </ItemGroup>
+
```

Comment:
Don't think this is needed. `#if NETSTANDARD2_1` should be enough

Reply:
Thx I just emoved the condition.